pkg/network/ebpf/c/protocols/protocol-classification-helpers.h

```diff
@@ -14,13 +14,52 @@ long bpf_skb_load_bytes(const void *skb, u32 offset, void *to, u32 len) {return
 #endif
 
 #define CHECK_PRELIMINARY_BUFFER_CONDITIONS(buf, buf_size, min_buff_size)   \
-        if (buf_size < min_buff_size) {                                     \
-            return false;                                                   \
-        }                                                                   \
+    if (buf_size < min_buff_size) {                                         \
```

Comment:
Remove spaces

Reply:
The spaces were moved before the if clause to be after it
using spaces instead of tabs